[2025-02-17T20:20:12.920+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-17T20:20:13.106+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: name_of_pipeline.save_data scheduled__2023-06-16T00:00:00+00:00 [queued]>
[2025-02-17T20:20:13.141+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: name_of_pipeline.save_data scheduled__2023-06-16T00:00:00+00:00 [queued]>
[2025-02-17T20:20:13.151+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-17T20:20:13.317+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): save_data> on 2023-06-16 00:00:00+00:00
[2025-02-17T20:20:13.490+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'name_of_pipeline', 'save_data', 'scheduled__2023-06-16T00:00:00+00:00', '--job-id', '803', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmpq0hr09pc']
[2025-02-17T20:20:13.502+0000] {standard_task_runner.py:105} INFO - Job 803: Subtask save_data
[2025-02-17T20:20:13.488+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2594) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-17T20:20:13.512+0000] {standard_task_runner.py:72} INFO - Started process 2636 to run task
[2025-02-17T20:20:14.206+0000] {task_command.py:467} INFO - Running <TaskInstance: name_of_pipeline.save_data scheduled__2023-06-16T00:00:00+00:00 [running]> on host f4228ad55513
[2025-02-17T20:20:17.244+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='name_of_pipeline' AIRFLOW_CTX_TASK_ID='save_data' AIRFLOW_CTX_EXECUTION_DATE='2023-06-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-16T00:00:00+00:00'
[2025-02-17T20:20:17.246+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-17T20:20:17.250+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-17T20:20:17.251+0000] {logging_mixin.py:190} INFO - Current task name:save_data state:running start_date:2025-02-17 20:20:13.107729+00:00
[2025-02-17T20:20:17.251+0000] {logging_mixin.py:190} INFO - Dag name:name_of_pipeline and current dag run status:running
[2025-02-17T20:20:17.257+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-17T20:20:17.258+0000] {pipeline.py:38} INFO - Сохраняем данные: LazySelectSequence([3 items])
[2025-02-17T20:20:17.294+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-17T20:20:17.378+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-17T20:20:17.399+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=name_of_pipeline, task_id=save_data, run_id=scheduled__2023-06-16T00:00:00+00:00, execution_date=20230616T000000, start_date=20250217T202013, end_date=20250217T202017
[2025-02-17T20:20:17.505+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-17T20:20:17.513+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-17T20:20:17.518+0000] {logging_mixin.py:190} INFO - Dag name:name_of_pipeline queued_at:2025-02-17 20:18:34.233616+00:00
[2025-02-17T20:20:17.519+0000] {logging_mixin.py:190} INFO - Task hostname:f4228ad55513 operator:PythonOperator
[2025-02-17T20:20:17.593+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-17T20:20:17.806+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-17T20:20:17.842+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
